src/parser/mod.rs

```diff
@@ -2160,6 +2103,43 @@ impl<'a> Parser<'a> {
         }))
     }
 
+    pub fn next_token_is_unit(&mut self) -> bool {
```

Comment:
Since this is a `pub fn` I think we should add some documentation or make it non `pub`

Reply:
done.